lib/linux_ccnt/aarch64_cycle_counter.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *    Copyright (c) 2011-2020 Seagate Technology LLC and/or its Affiliates
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ * Original author: Nalini <nalinikanta.jena@seagate.com>
+ * Original creation date: 31-May-2021
+ */
+
+#pragma once
+
+#ifndef __MOTR_LIB_LINUX_CCNT_A_H__
+#define __MOTR_LIB_LINUX_CCNT_A_H__
+
+/**
+ * @defgroup XXX
```

Comment:
Just remove this.

Reply:
the above macro can not be removed as it is part of coding style to include a header file , however i removed the other lines